spec/support/page_objects/teacher_interface/documents/document_form.rb

```diff
@@ -0,0 +1,10 @@
+module PageObjects
+  module TeacherInterface
+    class DocumentForm < SitePrism::Page
+      set_url "/teacher/"
```

Comment:
Is this URL right for this page? I reckon it's missing references to the document form.

Reply:
They aren't being used so can just be removed 👍